trogon/trogon.py

```diff
@@ -259,14 +275,18 @@ def run(
                     )
 
                     split_app_name = shlex.split(self.app_name)
-                    program_name = shlex.split(self.app_name)[0]
+                    program_name = split_app_name[0]
                     arguments = [*split_app_name, *self.post_run_command]
-                    os.execvp(program_name, arguments)
+                    # update PATH to include current working dir.
+                    env: dict[str, str] = os.environ.copy()
+                    env["PATH"] = os.pathsep.join([os.getcwd(), env["PATH"]])
+                    os.execvpe(program_name, arguments, env)
```

Comment:
Add the current working dir to PATH, so that a single executable script can be discovered by and invoked by Trogon.